README.md

```diff
@@ -1413,10 +1413,10 @@ If you add a dependency on a package with native code, you need to build the
 wheel manually:
 
 ```
-$ docker run -it -v ${azul_home}/:/root/azul python:3.6.8-stretch bash
+$ docker run -it -v ${azul_home}/:/root/azul python:3.8.2-stretch bash
 
 root:/# pip --version
-pip 19.1.1 from /usr/local/lib/python3.6/site-packages/pip (python 3.6)
+pip 19.1.1 from /usr/local/lib/python3.8/site-packages/pip (python 3.8)
```

Comment:
… and if this is still the actual output.

Reply:
Updated the output.